docs/en/engines/table-engines/integrations/materialized-postgresql.md

```diff
@@ -5,42 +5,52 @@ toc_title: MaterializedPostgreSQL
 
 # MaterializedPostgreSQL {#materialize-postgresql}
 
+Creates ClickHouse table with an initial data dump of PostgreSQL table and starts replication process, i.e. executes background job to apply new changes as they happen on PostgreSQL table in the remote PostgreSQL database.
```

Comment:
Will be good to add the same sentence (instead of the first one there) for database engine but substitute table with database and database tables.